src/SFA.DAS.AssessorService.Data.IntegrationTests/GetAssessmentOrganisationsByNameTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Application.Interfaces;
+using SFA.DAS.AssessorService.Data.IntegrationTests.Handlers;
+using SFA.DAS.AssessorService.Data.IntegrationTests.Models;
+using SFA.DAS.AssessorService.Data.IntegrationTests.Services;
+
+namespace SFA.DAS.AssessorService.Data.IntegrationTests
+{
+    public class GetAssessmentOrganisationsByNameTests : TestBase
+    {
+        private OrganisationModel _organisation1;
+        private OrganisationModel _organisation2;
+        private readonly DatabaseService _databaseService = new DatabaseService();
+        private RegisterQueryRepository _repository;
+        private string _orgId1;
```

Comment:
would fully qualify name _organisationId

Reply:
again, me being lazy. Will fix where I've done it.